install_config/install/disconnected_install.adoc

```diff
@@ -489,6 +489,46 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
 $REGISTRY:5000/openshift/webserver30-tomcat7-openshift:latest
 ----
 
+[[diconnected-configuring-registry-location]]
+=== Configuring a Registry Location
+
+If you are using an image registry other than the default at
+`registry.access.redhat.com`, specify the desired registry within the
+*_/etc/ansible/hosts_* file.
+
+----
+oreg_url=example.com/openshift3/ose-${component}:${version}
+openshift_examples_modify_imagestreams=true
+----
+
+If your registry is secure, you will also need to configure:
+----
+openshift_docker_additional_registries=$REGISTRY:5000
+openshift_docker_insecure_registries=$REGISTRY:5000
+----
```

Comment:
I do not think all the registry is using 5000 port, how about correct the above lines like this:
Depends on your registry configuration, you maybe also need to configure:
openshift_docker_additional_registries=example.com
openshift_docker_insecure_registries=example.com